testrace triggers the 5 minute timeout which is weird because it suggests a deadlock:

I0110 21:55:34.881802 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:34.883201 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56956" > attrs:<> 
E0110 21:55:34.886038 96093 gossip/gossip.go:185  different node IDs were set for the same gossip instance (1, 6)
I0110 21:55:34.886176 96093 gossip/gossip.go:193  setting node descriptor node_id:6 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:34.887620 96093 gossip/gossip.go:193  setting node descriptor node_id:6 address:<network_field:"" address_field:"" > attrs:<attrs:"us" attrs:"east" attrs:"gpu" attrs:"flaky" > 
I0110 21:55:34.888872 96093 gossip/gossip.go:193  setting node descriptor node_id:6 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:34.889900 96093 gossip/gossip.go:193  setting node descriptor node_id:6 address:<network_field:"" address_field:"" > attrs:<attrs:"nomatch" > 
I0110 21:55:34.891144 96093 gossip/gossip.go:193  setting node descriptor node_id:6 address:<network_field:"" address_field:"" > attrs:<attrs:"us" attrs:"east" attrs:"gpu" attrs:"flaky" attrs:"irrelevant" > 
I0110 21:55:34.892664 96093 gossip/gossip.go:193  setting node descriptor node_id:6 address:<network_field:"" address_field:"" > attrs:<attrs:"us" attrs:"east" attrs:"gpu" attrs:"flaky" attrs:"irrelevant" > 
I0110 21:55:34.894048 96093 gossip/gossip.go:193  setting node descriptor node_id:6 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:34.898284 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:34.898894 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56957" > attrs:<> 
E0110 21:55:34.900512 96093 gossip/gossip.go:185  different node IDs were set for the same gossip instance (1, 42)
I0110 21:55:34.900660 96093 gossip/gossip.go:193  setting node descriptor node_id:42 address:<network_field:"tcp" address_field:"foobar:1234" > attrs:<> 
I0110 21:55:34.904673 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:34.905291 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56958" > attrs:<> 
I0110 21:55:34.908990 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:34.909551 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56959" > attrs:<> 
I0110 21:55:35.142421 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:35.142928 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56969" > attrs:<> 
I0110 21:55:35.144628 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:35.145047 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56970" > attrs:<> 
I0110 21:55:35.146955 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:35.147466 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56971" > attrs:<> 
I0110 21:55:35.149699 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:35.150213 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56972" > attrs:<> 
I0110 21:55:35.394212 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:35.394688 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56981" > attrs:<> 
I0110 21:55:35.629580 96093 gossip/simulation/network.go:54  simulating gossip network with 3 nodes
I0110 21:55:35.630909 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56986" > attrs:<> 
I0110 21:55:35.632225 96093 gossip/gossip.go:193  setting node descriptor node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:56987" > attrs:<> 
I0110 21:55:35.633538 96093 gossip/gossip.go:738  starting client to 127.0.0.1:56986
I0110 21:55:35.633881 96093 gossip/gossip.go:193  setting node descriptor node_id:3 address:<network_field:"tcp" address_field:"127.0.0.1:56988" > attrs:<> 
I0110 21:55:35.635165 96093 gossip/gossip.go:738  starting client to 127.0.0.1:56986
I0110 21:55:35.637047 96093 gossip/server.go:160  node 1 returned 3 info(s) to node 2
I0110 21:55:35.638402 96093 gossip/server.go:140  node 1 received 3 (3 fresh) info(s) from node 2
I0110 21:55:35.638641 96093 gossip/server.go:211  @1: distantHops: 1 from 2
I0110 21:55:35.638818 96093 gossip/server.go:160  node 1 returned 6 info(s) to node 3
I0110 21:55:35.639285 96093 gossip/server.go:140  node 1 received 2 (2 fresh) info(s) from node 3
I0110 21:55:35.639383 96093 gossip/server.go:211  @1: distantHops: 1 from 2
I0110 21:55:35.643613 96093 gossip/simulation/network.go:153  gossip network simulation: total infos sent=9, received=5
I0110 21:55:35.644666 96093 gossip/client.go:77  closing client to 127.0.0.1:56986: gossip/client.go:251: client closed
I0110 21:55:35.644740 96093 gossip/client.go:77  closing client to 127.0.0.1:56986: read tcp 127.0.0.1:56990->127.0.0.1:56986: use of closed network connection
W0110 21:55:35.644832 96093 rpc/server.go:309  rpc: write response failed: write tcp 127.0.0.1:56986->127.0.0.1:56990: use of closed network connection
I0110 21:55:35.648045 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:35.648623 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56991" > attrs:<> 
I0110 21:55:35.650150 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:35.653863 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:35.654454 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56992" > attrs:<> 
E0110 21:55:35.655977 96093 gossip/gossip.go:185  different node IDs were set for the same gossip instance (1, 5)
I0110 21:55:35.656121 96093 gossip/gossip.go:193  setting node descriptor node_id:5 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:35.657976 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:35.658581 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56993" > attrs:<> 
I0110 21:55:35.669924 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:35.670671 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56994" > attrs:<> 
I0110 21:55:35.676914 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:35.677400 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56995" > attrs:<> 
I0110 21:55:35.679188 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:35.682454 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:35.683024 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56996" > attrs:<> 
I0110 21:55:35.684518 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:35.686580 96093 gossip/simulation/network.go:54  simulating gossip network with 1 nodes
I0110 21:55:35.687083 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:56997" > attrs:<> 
I0110 21:55:35.688782 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:35.696308 96093 kv/range_cache_test.go:48  getDescriptor: /Meta1/"aa"
I0110 21:55:35.696455 96093 kv/range_cache_test.go:48  getDescriptor: "aa"
I0110 21:55:35.697020 96093 kv/range_cache_test.go:136  doLookup: aa range_id:0 start_key:"a" end_key:"b" next_replica_id:0 
I0110 21:55:35.697170 96093 kv/range_cache_test.go:136  doLookup: ab range_id:0 start_key:"a" end_key:"b" next_replica_id:0 
I0110 21:55:35.697303 96093 kv/range_cache_test.go:136  doLookup: ba range_id:0 start_key:"b" end_key:"c" next_replica_id:0 
I0110 21:55:35.697438 96093 kv/range_cache_test.go:136  doLookup: cz range_id:0 start_key:"c" end_key:"d" next_replica_id:0 
I0110 21:55:35.697529 96093 kv/range_cache_test.go:48  getDescriptor: "d"
I0110 21:55:35.697747 96093 kv/range_cache_test.go:136  doLookup: d range_id:0 start_key:"d" end_key:"e" next_replica_id:0 
I0110 21:55:35.697878 96093 kv/range_cache_test.go:136  doLookup: fa range_id:0 start_key:"f" end_key:"g" next_replica_id:0 
I0110 21:55:35.697960 96093 kv/range_cache_test.go:48  getDescriptor: "ij"
I0110 21:55:35.698175 96093 kv/range_cache_test.go:136  doLookup: ij range_id:0 start_key:"i" end_key:"j" next_replica_id:0 
I0110 21:55:35.698304 96093 kv/range_cache_test.go:136  doLookup: jk range_id:0 start_key:"j" end_key:"k" next_replica_id:0 
I0110 21:55:35.698388 96093 kv/range_cache_test.go:48  getDescriptor: "pn"
I0110 21:55:35.698586 96093 kv/range_cache_test.go:136  doLookup: pn range_id:0 start_key:"p" end_key:"q" next_replica_id:0 
I0110 21:55:35.698680 96093 kv/range_cache_test.go:48  getDescriptor: /Meta1/"vu"
I0110 21:55:35.698860 96093 kv/range_cache_test.go:48  getDescriptor: "vu"
I0110 21:55:35.699051 96093 kv/range_cache_test.go:136  doLookup: vu range_id:0 start_key:"v" end_key:"w" next_replica_id:0 
I0110 21:55:35.699188 96093 kv/range_cache_test.go:136  doLookup: xx range_id:0 start_key:"x" end_key:"\377\377" next_replica_id:0 
W0110 21:55:35.699336 96093 kv/range_cache.go:208  compare-and-evict for key "da" with nil descriptor; clearing unconditionally
I0110 21:55:35.699504 96093 kv/range_cache_test.go:136  doLookup: fa range_id:0 start_key:"f" end_key:"g" next_replica_id:0 
I0110 21:55:35.699597 96093 kv/range_cache_test.go:48  getDescriptor: /Meta1/"da"
I0110 21:55:35.699755 96093 kv/range_cache_test.go:48  getDescriptor: "da"
I0110 21:55:35.699961 96093 kv/range_cache_test.go:136  doLookup: da range_id:0 start_key:"d" end_key:"e" next_replica_id:0 
I0110 21:55:35.700081 96093 kv/range_cache_test.go:136  doLookup: a range_id:0 start_key:"a" end_key:"b" next_replica_id:0 
I0110 21:55:35.700206 96093 kv/range_cache_test.go:136  doLookup: cz range_id:0 start_key:"c" end_key:"d" next_replica_id:0 
I0110 21:55:35.700323 96093 kv/range_cache_test.go:136  doLookup: cz range_id:0 start_key:"c" end_key:"d" next_replica_id:0 
I0110 21:55:35.700465 96093 kv/range_cache_test.go:48  getDescriptor: /Meta1/"cz"
I0110 21:55:35.700625 96093 kv/range_cache_test.go:48  getDescriptor: "cz"
I0110 21:55:35.700832 96093 kv/range_cache_test.go:136  doLookup: cz range_id:0 start_key:"c" end_key:"d" next_replica_id:0 
I0110 21:55:35.722821 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:35.723153 96093 kv/split_test.go:114  starting split at key "\"G\""...
I0110 21:55:35.724071 96093 storage/replica_command.go:1192  range 1: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:35.724696 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:35.727050 96093 storage/replica_command.go:1269  initiating a split of range=1 [/Min-/Max) at key "G"
I0110 21:55:35.733986 96093 kv/split_test.go:118  split at key "\"G\"" complete
I0110 21:55:35.734063 96093 kv/split_test.go:114  starting split at key "/Meta2/\"F\""...
I0110 21:55:35.734381 96093 storage/replica_command.go:1269  initiating a split of range=1 [/Min-"G") at key /Meta2/"F"
I0110 21:55:35.741690 96093 storage/replica_command.go:1192  range 2: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:35.742114 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
W0110 21:55:35.742874 96093 storage/stores.go:170  range not contained in one range: [/Min,"G\x00"), but have [/Min,"G")
I0110 21:55:35.749172 96093 kv/split_test.go:118  split at key "/Meta2/\"F\"" complete
I0110 21:55:35.749250 96093 kv/split_test.go:114  starting split at key "/Meta2/\"K\""...
I0110 21:55:35.750454 96093 storage/replica_command.go:1192  range 3: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
I0110 21:55:35.750737 96093 storage/replica_command.go:1269  initiating a split of range=3 [/Meta2/"F"-"G") at key /Meta2/"K"
W0110 21:55:35.751227 96093 storage/stores.go:170  range not contained in one range: [/Meta1/"K",/Meta2/"K\x00"), but have [/Min,/Meta2/"F")
I0110 21:55:35.764808 96093 kv/split_test.go:118  split at key "/Meta2/\"K\"" complete
I0110 21:55:35.764890 96093 kv/split_test.go:114  starting split at key "/Meta2/\"H\""...
I0110 21:55:35.765226 96093 storage/replica_command.go:1269  initiating a split of range=3 [/Meta2/"F"-/Meta2/"K") at key /Meta2/"H"
I0110 21:55:35.768669 96093 storage/replica_command.go:1192  range 4: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
W0110 21:55:35.778902 96093 storage/stores.go:170  range not contained in one range: [/Meta2/"F",/Meta2/"K\x00"), but have [/Meta2/"F",/Meta2/"K")
W0110 21:55:35.783712 96093 storage/replica.go:1686  failed to push during intent resolution: conflicting intents on [/Meta1/"H" /Meta1/"K"]: resolved? false
I0110 21:55:35.787300 96093 kv/split_test.go:118  split at key "/Meta2/\"H\"" complete
I0110 21:55:35.787859 96093 kv/split_test.go:123  mvcc scan should be clean: conflicting intents on [/Meta1/"H" /Meta1/"K"]: resolved? false
I0110 21:55:35.788252 96093 stopper.go:264  draining; tasks left:
1      storage/replica.go:1722
I0110 21:55:35.796798 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:35.797289 96093 storage/replica_command.go:1192  range 1: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:35.797686 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:35.800435 96093 kv/split_test.go:162  starting split at key "G"...
I0110 21:55:35.810134 96093 storage/replica_command.go:1269  initiating a split of range=1 [/Min-/Max) at key "G"
I0110 21:55:35.831894 96093 kv/split_test.go:166  split at key "G" complete
I0110 21:55:35.839018 96093 storage/replica_command.go:1192  range 2: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:35.840322 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:35.863818 96093 kv/split_test.go:162  starting split at key "R"...
I0110 21:55:35.864639 96093 storage/replica_command.go:1269  initiating a split of range=2 ["G"-/Max) at key "R"
I0110 21:55:35.923634 96093 kv/split_test.go:166  split at key "R" complete
I0110 21:55:35.923704 96093 kv/split_test.go:162  starting split at key "a"...
I0110 21:55:35.929637 96093 storage/replica_command.go:1192  range 3: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
W0110 21:55:35.931810 96093 storage/stores.go:170  range not contained in one range: ["MAPThbfdik","ogQjLovvQT\x00"), but have ["G","R")
I0110 21:55:35.932622 96093 storage/replica_command.go:1269  initiating a split of range=3 ["R"-/Max) at key "a"
E0110 21:55:35.935018 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:36.051794 96093 kv/split_test.go:166  split at key "a" complete
I0110 21:55:36.051878 96093 kv/split_test.go:162  starting split at key "l"...
I0110 21:55:36.058534 96093 storage/replica_command.go:1192  range 4: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:36.058874 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:36.059516 96093 storage/replica_command.go:1269  initiating a split of range=4 ["a"-/Max) at key "l"
I0110 21:55:36.151670 96093 kv/split_test.go:166  split at key "l" complete
I0110 21:55:36.151750 96093 kv/split_test.go:162  starting split at key "s"...
I0110 21:55:36.164440 96093 storage/replica_command.go:1192  range 5: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
I0110 21:55:36.165289 96093 storage/replica_command.go:1269  initiating a split of range=5 ["l"-/Max) at key "s"
E0110 21:55:36.166464 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
W0110 21:55:36.176842 96093 storage/stores.go:170  range not contained in one range: ["ahByaXmnJo","ukiNKcteaL\x00"), but have ["a","l")
W0110 21:55:36.194026 96093 storage/replica.go:1686  failed to push during intent resolution: conflicting intents on [/Meta2/Max]: resolved? false
W0110 21:55:36.197624 96093 storage/replica.go:1686  failed to push during intent resolution: conflicting intents on [/Meta2/Max]: resolved? false
I0110 21:55:36.260037 96093 kv/split_test.go:166  split at key "s" complete
I0110 21:55:36.266792 96093 storage/replica_command.go:1192  range 6: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:36.268305 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
W0110 21:55:36.271975 96093 storage/stores.go:170  range not contained in one range: ["lTIWdzyzWo","yLMqXUBeWo\x00"), but have ["l","s")
I0110 21:55:36.305329 96093 stopper.go:264  draining; tasks left:
2      storage/replica.go:1722
I0110 21:55:36.314998 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:36.315581 96093 storage/replica_command.go:1192  range 1: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:36.316033 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:36.649804 96093 storage/split_queue.go:113  splitting range=1 [/Min-/Max) size=275893 max=262144
I0110 21:55:36.651675 96093 storage/replica_command.go:1269  initiating a split of range=1 [/Min-/Max) at key "gBGjZfYbbH"
I0110 21:55:36.663087 96093 storage/replica_command.go:1192  range 2: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
I0110 21:55:36.878091 96093 storage/split_queue.go:113  splitting range=1 [/Min-"gBGjZfYbbH") size=276471 max=262144
I0110 21:55:36.879360 96093 storage/replica_command.go:1269  initiating a split of range=1 [/Min-"gBGjZfYbbH") at key "SPJNukhIKO"
W0110 21:55:36.886529 96093 storage/stores.go:170  range not contained in one range: [/Min,"gBGjZfYbbH\x00"), but have [/Min,"gBGjZfYbbH")
I0110 21:55:36.895196 96093 storage/replica_command.go:1192  range 3: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
I0110 21:55:36.942319 96093 storage/split_queue.go:113  splitting range=2 ["gBGjZfYbbH"-/Max) size=275534 max=262144
I0110 21:55:36.942834 96093 storage/replica_command.go:1269  initiating a split of range=2 ["gBGjZfYbbH"-/Max) at key "pdbStOLYgp"
W0110 21:55:36.962061 96093 storage/stores.go:170  range not contained in one range: [/Min,"SPJNukhIKO\x00"), but have [/Min,"SPJNukhIKO")
I0110 21:55:36.979035 96093 storage/replica_command.go:1192  range 4: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
I0110 21:55:37.193708 96093 storage/split_queue.go:113  splitting range=1 [/Min-"SPJNukhIKO") size=275957 max=262144
I0110 21:55:37.194395 96093 storage/replica_command.go:1269  initiating a split of range=1 [/Min-"SPJNukhIKO") at key "NFEEoBUSuH"
W0110 21:55:37.212427 96093 storage/stores.go:170  range not contained in one range: ["JqhMgbpmcK","NFEEoBUSuH\x00"), but have [/Min,"NFEEoBUSuH")
I0110 21:55:37.215149 96093 storage/replica_command.go:1192  range 5: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
W0110 21:55:37.216418 96093 storage/stores.go:170  range not contained in one range: ["mhmGkHHwYO","vLMMAKYLqf\x00"), but have ["gBGjZfYbbH","pdbStOLYgp")
I0110 21:55:37.398451 96093 storage/split_queue.go:113  splitting range=2 ["gBGjZfYbbH"-"pdbStOLYgp") size=262286 max=262144
I0110 21:55:37.399872 96093 storage/replica_command.go:1269  initiating a split of range=2 ["gBGjZfYbbH"-"pdbStOLYgp") at key "kimRoMuSQG"
I0110 21:55:37.416992 96093 storage/split_queue.go:113  splitting range=3 ["SPJNukhIKO"-"gBGjZfYbbH") size=278551 max=262144
I0110 21:55:37.417699 96093 storage/replica_command.go:1269  initiating a split of range=3 ["SPJNukhIKO"-"gBGjZfYbbH") at key "ZUXHBturgK"
I0110 21:55:37.423451 96093 storage/replica_command.go:1192  range 6: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
W0110 21:55:37.439717 96093 storage/stores.go:170  range not contained in one range: ["gBGjZfYbbH","kimRoMuSQG\x00"), but have ["gBGjZfYbbH","kimRoMuSQG")
I0110 21:55:37.526413 96093 kv/split_test.go:222  mvcc scan should be clean: conflicting intents on [/Meta2/"ZUXHBturgK" /Meta2/"gBGjZfYbbH" "oAiHnoPtCH" "pvOQkCnxmr"]: resolved? false
I0110 21:55:37.636117 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:37.636462 96093 kv/split_test.go:239  starting split at key "aa"...
I0110 21:55:37.651219 96093 storage/replica_command.go:1192  range 1: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:37.651669 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:37.653157 96093 storage/replica_command.go:1269  initiating a split of range=1 [/Min-/Max) at key "aa"
I0110 21:55:37.658995 96093 kv/split_test.go:243  split at key "aa" first time complete
I0110 21:55:37.660085 96093 storage/replica_command.go:1192  range 2: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:37.660930 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:37.681324 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:37.682380 96093 storage/replica_command.go:1192  range 1: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:37.682750 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:37.692036 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:37.693408 96093 storage/replica_command.go:1192  range 1: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:37.694556 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:37.708795 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:37.709283 96093 storage/replica_command.go:1192  range 1: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:37.709780 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:37.717526 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:37.719248 96093 storage/replica_command.go:1192  range 1: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:37.720106 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:37.741998 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:37.743344 96093 storage/replica_command.go:1192  range 1: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:37.749505 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
I0110 21:55:37.760851 96093 gossip/gossip.go:193  setting node descriptor node_id:1 address:<network_field:"" address_field:"" > attrs:<> 
I0110 21:55:37.772346 96093 storage/replica_command.go:1192  range 1: new leader lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC +1.000s
E0110 21:55:37.772874 96093 config/config.go:336  unable to determine largest reserved object ID from system config: descriptor table not found in system config of 0 values
W0110 21:55:37.779468 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.782890 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.784998 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.786444 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.788240 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.790674 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.792211 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.793859 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.796304 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.798586 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.800821 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.812610 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.814910 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.816490 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.818077 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.820208 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6
W0110 21:55:37.822805 96093 kv/txn_coord_sender.go:626  heartbeat to id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000000,6 orig=0.000000000,6 max=0.000000000,6 failed: retry txn id=c2464858 key="a" rw=true pri=70.25063162 iso=SERIALIZABLE stat=PENDING epo=0 ts=0.000000026,0 orig=0.000000000,6 max=0.000000000,6
panic: test timed out after 5m0s

goroutine 1365 [running]:
testing.startAlarm.func1()
	/private/var/folders/q8/bf_4b1ts2zj0l7b0p1dv36lr0000gp/T/workdir/go/src/testing/testing.go:703 +0x16b
created by time.goFunc
	/private/var/folders/q8/bf_4b1ts2zj0l7b0p1dv36lr0000gp/T/workdir/go/src/time/sleep.go:129 +0x6e

goroutine 1 [chan receive, 4 minutes]:
testing.RunTests(0x55a1f90, 0x59a2760, 0x4d, 0x4d, 0x521c801)
	/private/var/folders/q8/bf_4b1ts2zj0l7b0p1dv36lr0000gp/T/workdir/go/src/testing/testing.go:562 +0xafa
testing.(*M).Run(0xc82041ff10, 0x72015b5a198d5e90)
	/private/var/folders/q8/bf_4b1ts2zj0l7b0p1dv36lr0000gp/T/workdir/go/src/testing/testing.go:494 +0xe5
github.com/cockroachdb/cockroach/util/leaktest.TestMainWithLeakCheck(0xc82041ff10)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/util/leaktest/leaktest.go:37 +0x2f
github.com/cockroachdb/cockroach/kv_test.TestMain(0xc82041ff10)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/kv/main_test.go:34 +0x2f
main.main()
	github.com/cockroachdb/cockroach/kv/_test/_testmain.go:210 +0x20a

goroutine 17 [syscall, 4 minutes, locked to thread]:
runtime.goexit()
	/private/var/folders/q8/bf_4b1ts2zj0l7b0p1dv36lr0000gp/T/workdir/go/src/runtime/asm_amd64.s:1696 +0x1

goroutine 5 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x66775a0)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:1029 +0x76
created by github.com/cockroachdb/cockroach/util/log.init.1
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:610 +0x117

goroutine 1877 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*Replica).addWriteCmd(0xc820150380, 0x7842e20, 0xc8202a1bc0, 0x1a, 0x6, 0x100000001, 0x1, 0x1, 0x0, 0xc8203ea6c0, ...)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/storage/replica.go:970 +0x7be
github.com/cockroachdb/cockroach/storage.(*Replica).Send(0xc820150380, 0x7842e20, 0xc8202a1bc0, 0x1a, 0x6, 0x100000001, 0x1, 0x1, 0x0, 0xc8203ea6c0, ...)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/storage/replica.go:672 +0x68d
github.com/cockroachdb/cockroach/storage.(*Store).Send(0xc8202b2000, 0x7842e20, 0xc8202a1bc0, 0x1a, 0x6, 0x100000001, 0x1, 0x1, 0x0, 0xc8203ea6c0, ...)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/storage/store.go:1277 +0xda0
github.com/cockroachdb/cockroach/storage.(*Stores).Send(0xc8203ee160, 0x7842e20, 0xc8202a1b60, 0x1a, 0x6, 0x100000001, 0x1, 0x1, 0x0, 0xc8203ea6c0, ...)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/storage/stores.go:150 +0x445
github.com/cockroachdb/cockroach/kv.(*LocalTestCluster).Start.func1(0x1, 0x1, 0x2540be400, 0x12a05f200, 0xc8204fc1c0, 0x533a340, 0xa, 0xc8205551d0, 0x1, 0x1, ...)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/kv/local_test_cluster.go:88 +0x1f9
github.com/cockroachdb/cockroach/kv.(*DistSender).sendRPC(0xc820014460, 0xc8204fc1c0, 0x1, 0xc820a889c0, 0x1, 0x1, 0x1, 0x1a, 0x6, 0x0, ...)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/kv/dist_sender.go:373 +0x7b9
github.com/cockroachdb/cockroach/kv.(*DistSender).sendSingleRange(0xc820014460, 0xc8204fc1c0, 0x1a, 0x6, 0x0, 0x0, 0x0, 0x0, 0xc8203ea6c0, 0x0, ...)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/kv/dist_sender.go:451 +0x279
github.com/cockroachdb/cockroach/kv.(*DistSender).sendChunk.func1(0xc8205b94e8, 0xc8205b9250, 0xc8205b9890, 0xc820014460, 0xc8204fc1c0, 0x2, 0x8)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/kv/dist_sender.go:628 +0x59e
github.com/cockroachdb/cockroach/kv.(*DistSender).sendChunk(0xc820014460, 0x7842e20, 0xc8203915f0, 0x1a, 0x6, 0x0, 0x0, 0x0, 0x0, 0xc8203ea6c0, ...)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/kv/dist_sender.go:629 +0x16c1
github.com/cockroachdb/cockroach/kv.(*DistSender).Send(0xc820014460, 0x7842e20, 0xc8203915f0, 0x1a, 0x6, 0x0, 0x0, 0x0, 0x0, 0xc8203ea6c0, ...)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/kv/dist_sender.go:502 +0x30d
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).heartbeat(0xc82050dd60, 0xc8202b49c0, 0x10, 0xc8204fc1c0, 0x7842e20, 0xc8203915f0, 0x1)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:620 +0x623
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).heartbeatLoop(0xc82050dd60, 0xc8202b49c0, 0x10)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:571 +0x64e
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).updateState.func1.1.1()
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:759 +0x4d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820014150, 0xc82036e940)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1846 [semacquire, 4 minutes]:
sync.runtime_Semacquire(0xc820014174)
	/private/var/folders/q8/bf_4b1ts2zj0l7b0p1dv36lr0000gp/T/workdir/go/src/runtime/sema.go:43 +0x26
sync.(*WaitGroup).Wait(0xc820014168)
	/private/var/folders/q8/bf_4b1ts2zj0l7b0p1dv36lr0000gp/T/workdir/go/src/sync/waitgroup.go:126 +0x118
github.com/cockroachdb/cockroach/util/stop.(*Stopper).Stop(0xc820014150)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:214 +0x22b
github.com/cockroachdb/cockroach/kv.(*LocalTestCluster).Stop(0xc8200140e0)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/kv/local_test_cluster.go:147 +0xd4
github.com/cockroachdb/cockroach/kv.TestTxnCoordSenderHeartbeat(0xc82009a000)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender_test.go:282 +0x79b
testing.tRunner(0xc82009a000, 0x59a2aa8)
	/private/var/folders/q8/bf_4b1ts2zj0l7b0p1dv36lr0000gp/T/workdir/go/src/testing/testing.go:456 +0xdd
created by testing.RunTests
	/private/var/folders/q8/bf_4b1ts2zj0l7b0p1dv36lr0000gp/T/workdir/go/src/testing/testing.go:561 +0xaa4

goroutine 1847 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8201a4280)
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/Users/vivek1/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x599
FAIL	github.com/cockroachdb/cockroach/kv	300.069s
ok  	github.com/cockroachdb/cockroach/roachpb	1.272s
ok  	github.com/cockroachdb/cockroach/rpc	8.326s
ok  	github.com/cockroachdb/cockroach/rpc/codec	1.037s
?   	github.com/cockroachdb/cockroach/rpc/codec/message	[no test files]
?   	github.com/cockroachdb/cockroach/rpc/codec/wire	[no test files]
ok  	github.com/cockroachdb/cockroach/security	1.967s
?   	github.com/cockroachdb/cockroach/security/securitytest	[no test files]
ok  	github.com/cockroachdb/cockroach/server	14.959s
ok  	github.com/cockroachdb/cockroach/server/status	1.409s
ok  	github.com/cockroachdb/cockroach/sql	26.363s
ok  	github.com/cockroachdb/cockroach/sql/driver	10.582s
ok  	github.com/cockroachdb/cockroach/sql/parser	1.221s
ok  	github.com/cockroachdb/cockroach/sql/pgwire	1.051s
ok  	github.com/cockroachdb/cockroach/sql/privilege	1.034s
ok  	github.com/cockroachdb/cockroach/storage	14.897s
ok  	github.com/cockroachdb/cockroach/storage/engine	1.665s
?   	github.com/cockroachdb/cockroach/storage/engine/rocksdb	[no test files]
?   	github.com/cockroachdb/cockroach/storage/simulation	[no test files]
?   	github.com/cockroachdb/cockroach/testutils	[no test files]
?   	github.com/cockroachdb/cockroach/testutils/gossiputil	[no test files]
ok  	github.com/cockroachdb/cockroach/ts	1.359s
?   	github.com/cockroachdb/cockroach/ui	[no test files]
ok  	github.com/cockroachdb/cockroach/util	1.045s
ok  	github.com/cockroachdb/cockroach/util/cache	1.020s
ok  	github.com/cockroachdb/cockroach/util/caller	1.020s
ok  	github.com/cockroachdb/cockroach/util/encoding	1.028s
ok  	github.com/cockroachdb/cockroach/util/hlc	1.021s
?   	github.com/cockroachdb/cockroach/util/leaktest	[no test files]
ok  	github.com/cockroachdb/cockroach/util/log	2.052s
?   	github.com/cockroachdb/cockroach/util/log/logflags	[no test files]
ok  	github.com/cockroachdb/cockroach/util/metric	1.030s
ok  	github.com/cockroachdb/cockroach/util/randutil	1.028s
ok  	github.com/cockroachdb/cockroach/util/retry	1.020s
ok  	github.com/cockroachdb/cockroach/util/stop	1.054s
ok  	github.com/cockroachdb/cockroach/util/tracer	1.034s
ok  	github.com/cockroachdb/cockroach/util/uuid	1.017s
make: *** [testrace] Error 1

--- a/kv/txn_coord_sender.go
+++ b/kv/txn_coord_sender.go
@@ -574,6 +574,9 @@ func (tc *TxnCoordSender) heartbeatLoop(id string) {
 		case <-closer:
 			// Transaction finished normally.
 			return
+
+		case <-tc.stopper.ShouldDrain():
+			return
 		}
 	}
 }

--- a/kv/txn_coord_sender_test.go
+++ b/kv/txn_coord_sender_test.go
@@ -18,7 +18,6 @@ package kv
 
 import (
 	"bytes"
-	"errors"
 	"fmt"
 	"reflect"
 	"strconv"
@@ -592,54 +591,6 @@ func TestTxnCoordSenderTxnUpdatedOnError(t *testing.T) {
 				i, test.nodeSeen, nodes)
 		}
 	}
-}
-
-// TestTxnDrainingNode tests that pending transactions tasks' intents
-// are resolved if they commit while draining, and that a
-// NodeUnavailableError is received when attempting to run a new
-// transaction on a draining node.
-func TestTxnDrainingNode(t *testing.T) {
-	defer leaktest.AfterTest(t)
-	s := createTestDB(t)
-
-	done := make(chan struct{})
-	// Dummy task that keeps the node in draining state.
-	if !s.Stopper.RunAsyncTask(func() {
-		<-done
-	}) {
-		t.Fatal("stopper draining prematurely")
-	}
-
-	key := roachpb.Key("a")
-	txn := client.NewTxn(*s.DB)
-	// Begin before draining.
-	if err := txn.Put(key, []byte("value")); err != nil {
-		t.Fatal(err)
-	}
-	go func() {
-		s.Stopper.Stop()
-	}()
-	util.SucceedsWithin(t, time.Second, func() error {
-		if s.Stopper.RunTask(func() {}) {
-			return errors.New("stopper not yet draining")
-		}
-		return nil
-	})
-	// Commit after draining.
-	if err := txn.Commit(); err != nil {
-		t.Fatal(err)
-	}
-	verifyCleanup(key, s.Sender, s.Eng, t) // make sure intent gets resolved
-
-	// Attempt to start another transaction, but it should be too late.
-	txn2 := client.NewTxn(*s.DB)
-	err := txn2.Put(key, []byte("value"))
-	if _, ok := err.(*roachpb.NodeUnavailableError); !ok {
-		teardownHeartbeats(s.Sender)
-		t.Fatal(err)
-	}
-	close(done)
-	<-s.Stopper.IsStopped()
 }
 
 // TestTxnMultipleCoord checks that a coordinator uses the Writing flag to

